glue/plugins/scatter_group/__init__.py

```diff
@@ -0,0 +1,10 @@
+def setup():
+    from ...logger import logger
```

Comment:
@astrofrog this logger module is new to me -- is that something you added? Before I was calling logging.getLogger(__name__) to organize loggers into per-module hierarchies. Is that not needed anymore?